web/src/views/repo/pipeline/PipelineWrapper.vue

```diff
@@ -31,6 +31,17 @@
             :is-loading="isRestartingPipeline"
             @click="restartPipeline"
           />
+          <Button
+            v-if="pipeline.status === 'success'"
+            class="flex-shrink-0"
+            :text="$t('repo.pipeline.actions.deploy')"
+            @click="showDeployPipelinePopup = true"
+          />
+          <DeployPipelinePopup
+            :id="pipelineId"
```

Comment:
```suggestion
            :pipelineId="pipelineId"
```

Reply:
If I rename it, should it be `pipelineId` which is not equla to `pipeline.id` or should it be `pipelineNumber`?

It is the number of the pipeline execution which was build number some time ago.

Reply:
Good point `pipelineNumber` would be better.

Reply:
It is `pipelineNumber` now